vispy/plot/__init__.py

```diff
@@ -31,7 +31,6 @@
 
 """
 
-from .fig import Fig
-from .plotwidget import PlotWidget
-
-__all__ = ['Fig', 'PlotWidget']
+from .fig import Fig  # noqa
+from .plotwidget import PlotWidget  # noqa
+from ..scene import *  # noqa
```

Comment:
I'm not sure about this one -- do we need to populate this namespace with everything from `scene`?